internal/ihop/definition.go

```diff
@@ -151,6 +162,14 @@ func NewDefinitionFromFile(path string, unbuffered bool, secrets ...string) (Def
 		definition.Run.Shell = "/sbin/nologin"
 	}
 
+	if definition.SupportURL == "" {
+		definition.SupportURL = fmt.Sprintf("%s/blob/main/README.md", strings.TrimSuffix(definition.Homepage, "/"))
+	}
+
+	if definition.BugReportURL == "" {
+		definition.BugReportURL = fmt.Sprintf("%s/issues/new", strings.TrimSuffix(definition.Homepage, "/"))
+	}
+
```

Comment:
Its maybe worth doing some kind of match on the homepage URL to check that its pointing to GitHub. If we move this code earlier (prior to the required fields check), then we can only these values if they are empty and matching `github.com`, and we can fail the parse call otherwise if they are truly required.

Reply:
I like the proposed behavior of having `name` and `homepage` as required and `support-url` and `bug-report-url` as optional in the `stack.toml` file, completing them based on `homepage` if they are omitted.

I also have a slight preference for checking that the `homepage` contains `github.com` before filling in the blanks. It seems easy to miss this when creating a stack whose `homepage` is not on github. I don't expect folks to read the code/docs before making a stack - I expect most people will copy an example `stack.toml` file from somewhere, and most of those existing stacks will have a `homepage` on github.